plugins/cue_points/quiz/lib/model/kQuizManager.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @package plugins.quiz
+ */
+class kQuizManager implements kObjectAddedEventConsumer, kObjectChangedEventConsumer
```

Comment:
I would delete this entire event consumer and implement it in pastSave and postInsert on the AnswerCuePoint.
Event consumers are needed only when one plugin want to react to core or different plugin actions.